### DISCUSSION TITLE
Thoughts on Nerdfont

### DISCUSSION THREAD

**Discussion by Bas950** [JanDeDobbeleer/oh-my-posh#2136] [Ideas]
I wanted to add support for pnpm versions and some others, but I noticed Nerdfont doesn't have those icons...
Afterwards I looked into adding icons to nerdfont, but all the sources for icons they use are either heavily regulated or inactive.
I saw the current icon for Yarn is F61A which in nerdfont is the `nf-mdi-cat` icon (see attachment), so right now we aren't even using the 'correct' icon for it.

So I wanted to hear your thoughts on creating an oh-my-posh-fonts pack which includes icons you would use in a terminal, since a lot of icons in nerdfonts will never/hardly be used... (Don't think anyone will use the many car icons there are in nerdfonts in a terminal)

![image](https://user-images.githubusercontent.com/34792838/164915020-dc3a20d1-e50f-408c-81d3-f0451420b2e9.png)

**Comment by JanDeDobbeleer** [comment]
@Bas950 that's an interesting question. I don't see how we would maintain this tbh. I'm maybe even more inclined to see if we could have conversation with @cinnamon-msft on Cascadia Code PL and patch that one with the right icons (not necessarily NF based).

**Reply by Bas950** [reply]
Yea patch fonts like Cascadia Code, Fira Code etc.
But it would definitely be a bit difficult to maintain, but could also be a fun challenge.

**Reply by JanDeDobbeleer** [reply]
I have more than enough challenges myself, but feel free to take it 😁

**Reply by Bas950** [reply]
Same at the moment, but I will look into it if I get some spare time!
